### DISCUSSION TITLE
Using your classes w1/2 etc. in angular

### DISCUSSION THREAD

**Discussion by Disane87** [tailwindlabs/tailwindcss#1533] [General]
Hey guys, I really love your peojcts but now I'm having serious problems with some particular classes, like the half width classes (all clases with `/`). In Angular I can work with classes like this 
```html
<div class="flex" [class.w-full]="full">
``` 

But when I want to use 
```html
<div class="flex" [class.w1/2]="full">
``` 

Angular compiler is complaing:
>ERROR in Errors parsing template: Unexpected closing tag "a". It may happen when the tag has already been closed by another tag. For more info see https://www.w3.org/TR/html5/syntax.html#closing-elements-that-have-implied-end-tags ("y-600 mt-3">
                                                        <a class="btn" href="mailto:hi@la-vielie.de" [class.w1/2]="true">hi@la-vielie.de[ERROR ->]</a>
                                                </div>
                                        </div>

Is there anybody who encounters this problem too and have a fix for that?

**Comment by TrentFoulds** [comment]
So do you think using getters that produces a value based on class fields is a bad practice? If it is what would be a good practice to achieve such?
[Macys Insite](https://www.macysinsite.us/)
Say for example i have two fields Name and Surname. Im used to classes to I usually create a FullName getter that returns the two in one string. What would be the better solution if interfaces are used?

**Comment by Disane87** [comment]
> So do you think using getters that produces a value based on class fields is a bad practice? If it is what would be a good practice to achieve such?
> 
> 
> 
> Say for example i have two fields Name and Surname. Im used to classes to I usually create a FullName getter that returns the two in one string. What would be the better solution if interfaces are used?

I don't know what your reply has something to do with my problem?!
